### PR TITLE
OoB docs update

### PR DESCRIPTION
**Pull request by LDong-Arm**
This PR
* adds a README to the BLE_GattServer example
* removes outdated information in the top-level README

### CODE REVIEW

**Comment by LDong-Arm** [comment]
Fixes #299

File: BLE_GattServer/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# BLE Gatt Server example
+
+This application demonstrates detailed uses of the GattServer APIs.
+
+It starts by advertising to its environment with the device name "GattServer". Once you connect to the device with a BLE scanner on your phone, the scanner shows a service with three characteristics - each representing the hour, minute and second of a clock.
+
+To see the clock values updating, subscript to the service using the "Enable CCCDs" (or similar) option provided by the scanner. Now the values get updated once a second.
```
Comment (paul-szczepanek-arm) [inline BLE_GattServer/README.md:7]:
```suggestion
To see the clock values updating subscribe to the service using the "Enable CCCDs" (or similar) option provided by the scanner. Now the values get updated once a second.
```

File: BLE_GattServer/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+# Running the application
+
+## Requirements
+
+You may use a generic BLE scanners:
```
Comment (paul-szczepanek-arm) [inline BLE_GattServer/README.md:13]:
```suggestion
You may use any BLE scanner, for example:
```

**Comment by LDong-Arm** [comment]
@paul-szczepanek-arm Thanks for the suggestions.

Guess we can get it in?

**Comment by paul-szczepanek-arm** [comment]
only docs updated, merged

**Pull request merged**